rbac/providers/ldap/ldap_validator.py

```diff
@@ -12,5 +12,74 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ------------------------------------------------------------------------------
+import sys
+import logging
+from rbac.providers.common.expected_errors import ValidationException
 
-# TODO: Add validation for LDAP sync payloads
+LOGGER = logging.getLogger(__name__)
+LOGGER.level = logging.DEBUG
```

Comment:
Should we set the logging level to INFO?

Reply:
It has been updated to INFO logging level. I copied this from expected_errors.py and I have changed it on that file too.

Reply:
awesome! sounds good, thanks!